api/src/main/java/org/pmiops/workbench/vwb/usermanager/VwbUserManagerClient.java

```diff
@@ -87,4 +88,52 @@ private void updateGroupMembership(String groupName, String email, SetAccessOper
           return null;
         });
   }
+
+  public PodDescription createInitialCreditsPodForUser(String email) {
+    String organizationId = workbenchConfigProvider.get().vwb.organizationId;
+    String initialCreditBillingAccount = workbenchConfigProvider.get().billing.accountId;
+    logger.info("Creating pod for user in VWB with email {}", email);
+    return vwbUserManagerRetryHandler.run(
+        context ->
+            podApiProvider
+                .get()
+                .createPod(
+                    new PodCreateRequest()
+                        .userFacingId("user-pod-" + email.substring(0, email.indexOf('@')))
```

Comment:
I chose this as the name of the pod to be unique per user.

Reply:
do you think better to have "initial-credit-". Also note, many email has dot. dot may not work well in some systems (or that is not an issue)? Another thing is "initial-credit-{userid}" ?

Reply:
I didn't want to put initial-credit in the name, because we'll give the user pod manager on it, and therefore the use will change it to BYO billing, so it would be confusing.